### PR TITLE
2024Q1/Status/intro.adoc: Add file

### PR DESCRIPTION
**Pull request by lsalvadore**

### CODE REVIEW

**Comment by lsalvadore** [comment]
I have just noticed that I have inverted Status and 2024Q1 in the commit summary. I will fix it on merging.

File: website/content/en/status/report-2024-01-2024-03/intro.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Here is the first 2024 status report, with 22 entries.
+
+The New Year brings us many new interesting projects, such as the new `libsys` that separates system calls from `libc` and `libpthread` or work on a graphical installer for FreeBSD, which will help making our OS more user-friendly.
+Of course, the usual projects keeps going on, such as the work on cloud-init, openstack or the GCC ports.
```
Comment (pauamma) [inline website/content/en/status/report-2024-01-2024-03/intro.adoc:4]: s/keeps/keep/
s/openstack/Openstack,/ (capitalization and serial comma)

File: website/content/en/status/report-2024-01-2024-03/intro.adoc [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+As usual our main teams share with us their progress.
```
Comment (pauamma) [inline website/content/en/status/report-2024-01-2024-03/intro.adoc:5]: "...share their progress with us", I think.

**Comment by lsalvadore** [comment]
Merged, thanks!

**Pull request closed without merging**